src/Template.php

```diff
@@ -34,10 +34,18 @@ public function render(array $data = []): string
             View::share('pages', $data['pages']);
             View::share('page', $data['page']);
 
-            return View::file($this->file(), $data)->render();
+            $html = View::file($this->file(), $data)->render();
+        } else {
+            $html = Tpl::load($this->file(), $data);
+        }
+
+        $callback = option('leitsch.blade.callback', null);
+
+        if ($callback && is_callable($callback)) {
+            return $callback($html);
```

Comment:
We should use a hook instead of the callback. Is there anything against using a hook from your point of view?

As name, we could use `blade.render:after`

https://getkirby.com/docs/reference/plugins/extensions/hooks#creating-your-own-hooks

Reply:
Mkay